xskillscore/core/comparative.py

```diff
@@ -192,19 +196,21 @@ def _categorical_metric(observations, forecasts, dim):
     return significantly_different, walk, confidence
 
 
-def mae_test(
-    forecasts1,
-    forecasts2,
-    observations=None,
-    dim=[],
-    time_dim="time",
-    alpha=0.05,
-):
+def significance_test(
+    forecasts1: XArray,
```

Comment:
how does this typing show up in the docs?

May be worth explicitly using `Union[xr.Dataset, xr.DataArray]` instead of `XArray`

Reply:
I will change it.

Reply:
looks like this now: 
![image](https://user-images.githubusercontent.com/12237157/118102135-afe10680-b3d8-11eb-9be6-9084dcc06aff.png)

Reply:
So, it's not showing the types, because this is what it should look like, this is screen short from my packages docs:
![image](https://user-images.githubusercontent.com/13016644/118141267-ca15e700-b3ce-11eb-8c9c-5d83a906e1be.png)

Reply:
I think you need this extension in `conf.py`: `sphinx.ext.autodoc.typehints`

Reply:
This is the relevant part from sphinx's doc (I use `description` for my package):
![image](https://user-images.githubusercontent.com/13016644/118145843-a5703e00-b3d3-11eb-8161-ec9b19d15fcf.png)

Reply:
please add. we were anyways considering using them as you saw

Reply:
DO you want me to add it in this PR or you will do it yourself later?

Reply:
> DO you want me to add it in this PR or you will do it yourself later?

Yes please. and if you original `XArray` variable is shown correctly you can use that. Sorry for the back-and-forth suggestion here. That for also working on another issue in the same PR.

Reply:
Sure, will do that. Happy to contribute!

Reply:
I tried to add this, but for some reason it's not working. I couldn't figure out the issue. It doesn't throw any errors or warnings and builds successfully but types won't show up for the function.